client/src/shared/components/Layout-learning/Layout-learning.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
```

Comment:
- file structure is incorrect. pls inquire to members the proper folder structure.
![image](https://user-images.githubusercontent.com/73863685/222644670-4418dcd4-d49a-4657-a804-db43ed68d506.png)